lib/dotcom_web/components/trip_planner/itinerary_summary.ex

```diff
@@ -23,18 +23,23 @@ defmodule DotcomWeb.Components.TripPlanner.ItinerarySummary do
         <% end %>
       </div>
       <div class="flex flex-wrap gap-1 items-center mb-3 text-sm text-grey-dark">
-        <div :if={@summary.accessible?} class="inline-flex items-center gap-0.5">
-          <.icon type="icon-svg" name="icon-accessible-small" class="h-3 w-3 mr-0.5" /> Accessible
+        <div class="inline-flex items-center gap-0.5">
+          <%= if @summary.accessible? do %>
+            <.icon type="icon-svg" name="icon-accessible-small" class="h-3 w-3 mr-0.5" /> Accessible
+          <% else %>
+            <.icon type="icon-svg" name="icon-not-accessible-small" class="h-3 w-3 mr-0.5" /> May not be accessible
+          <% end %>
           <.icon name="circle" class="h-0.5 w-0.5 mx-1" />
         </div>
         <div class="inline-flex items-center gap-0.5">
           <.icon name="person-walking" class="h-3 w-3" />
           {@summary.walk_distance} mi
         </div>
-        <div :if={@summary.total_cost > 0} class="inline-flex items-center gap-0.5">
+        <% price = Fares.Format.price(@summary.total_cost) %>
+        <div :if={price != ""} class="inline-flex items-center gap-0.5">
           <.icon name="circle" class="h-0.5 w-0.5 mx-1" />
           <.icon name="wallet" class="h-3 w-3" />
-          {Fares.Format.price(@summary.total_cost)}
+          {price}
```

Comment:
📓 If the price is `nil` then "" will be returned. If the price is 0 then "Free" will be returned. If we get "" then we don't show the element.